diff -ru a/drivers/modem/Kconfig b/drivers/modem/Kconfig
--- a/drivers/modem/Kconfig	2018-02-17 14:51:57.705103361 +0300
+++ b/drivers/modem/Kconfig	2018-02-17 12:03:12.929645868 +0300
@@ -26,7 +26,7 @@
 source "drivers/modem/efuse/Kconfig"
 source "drivers/modem/balong_oam_ps/Kconfig"
 source "drivers/modem/reset/Kconfig"
-source "drivers/modem/reset_hifi/Kconfig"
+#source "drivers/modem/reset_hifi/Kconfig"
 source "drivers/modem/lowpower_mntn/Kconfig"
 source "drivers/modem/temperature/Kconfig"
 source "drivers/modem/dialup_hsuart/Kconfig"
Only in b/include/linux: compiler-gcc7.h
diff -ru a/include/linux/netfilter/Kbuild b/include/linux/netfilter/Kbuild
--- a/include/linux/netfilter/Kbuild	2018-02-17 14:51:58.712088698 +0300
+++ b/include/linux/netfilter/Kbuild	2018-02-17 12:05:59.329215198 +0300
@@ -75,6 +75,4 @@
 header-y += xt_tcpudp.h
 header-y += xt_time.h
 header-y += xt_u32.h
-/*BEGIN DTS2013061307804    s00201037 2013/07/03 added*/
 header-y += xt_dns_flt.h
-/*END DTS2013061307804    s00201037 2013/07/03 added*/
\ No newline at end of file
diff -ru a/include/linux/netfilter_ipv4/Kbuild b/include/linux/netfilter_ipv4/Kbuild
--- a/include/linux/netfilter_ipv4/Kbuild	2018-02-17 14:51:58.716088640 +0300
+++ b/include/linux/netfilter_ipv4/Kbuild	2018-02-17 12:06:12.250026511 +0300
@@ -10,7 +10,5 @@
 header-y += ipt_ah.h
 header-y += ipt_ecn.h
 header-y += ipt_ttl.h
-/*BEGIN DTS2013061307804    s00201037 2013/07/03 added*/
 header-y += ipt_TRIGGER.h
 header-y += ipt_webstr.h
-/*END DTS2013061307804    s00201037 2013/07/03 added*/
\ No newline at end of file
